iphone/Classes/TiUIWindowProxy.m

```diff
@@ -363,6 +363,23 @@ -(void)setBarColor:(id)colorString
     }
 }
 
+-(void)setStatusBarBackgroundColor:(id)colorString
+{
+    ENSURE_UI_THREAD(setStatusBarBackgroundColor, colorString);
+    NSString *color = [TiUtils stringValue:colorString];
+    [self replaceValue:color forKey:@"statusBarBackgroundColor" notification:NO];
+    if (controller != nil && [controller navigationController] != nil)
+    {
+        if (statusbarBg == nil) {
+            statusbarBg = [[UIView alloc] initWithFrame:[UIApplication sharedApplication].statusBarFrame];
+            [[controller navigationController].view addSubview:statusbarBg];
+        }
+        TiColor* newColor = [TiUtils colorValue:color];
+        statusbarBg.backgroundColor = [newColor _color];
+    }
+}
```

Comment:
Not sure if that's the best way to go. I like the style proposed here: http://stackoverflow.com/a/34658477/5537752

Also, `statusbarBg` should be released from memory every time it is set again, as well as in the `dealloc` method to clean it from memory after the gets GC'd.

Please try the above-mentioned way to do it.